src/SFML/Graphics/Texture.cpp

```diff
@@ -265,24 +265,28 @@ bool Texture::create(const Vector2u& size)
 ////////////////////////////////////////////////////////////
 bool Texture::loadFromFile(const std::filesystem::path& filename, const IntRect& area)
 {
-    Image image;
-    return image.loadFromFile(filename) && loadFromImage(image, area);
+    const auto image = sf::Image::loadFromFile(filename);
+    return loadFromImage(*image, area);
```

Comment:
Uh, can't `image` fail to load here? Shouldn't we check and propagate a possible error like so?

```suggestion
    const auto image = sf::Image::loadFromFile(filename);
    
    if (!image.has_value())
        return false;
    
    return loadFromImage(*image, area);
```